### DISCUSSION TITLE
TTF fonts are not loading in SDK 42

### DISCUSSION THREAD

**Discussion by dashcraft** [expo/expo#14149] [Archived: Questions] [answered]
I'm using sdk 42 for a production application, however all my ttf fonts are now broken.

Is anything else experiencing this or has found a work around?

There's already a month old issue out about this:
https://github.com/expo/expo/issues/13839

**Comment by EvanBacon** [comment, marked as answer]
[Answered here](https://github.com/expo/expo/issues/13839#issuecomment-912816569). 
TL;DR: Use `yarn` instead of `npm install` or highly frustrating / silly issues like this can occur.
